components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml

```diff
@@ -257,6 +257,67 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </date_of_death_entered>
+    <delivery_mode>
+      <cache>0</cache>
+      <classname/>
+      <customDisplay>{{velocity}}
+#set ($order = [
```

Comment:
Most of them are delivery complications, not methods.
On Jun 10, 2015 10:16 AM, "Matthew Shepherd" <notifications@github.com>
wrote:

> In
> components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml
> <https://github.com/phenotips/phenotips/pull/1698#discussion_r32121183>:
>
> > @@ -257,6 +257,67 @@
> >        <validationRegExp/>
> >        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
> >      </date_of_death_entered>
> > +    <delivery_mode>
> > +      <cache>0</cache>
> > +      <classname/>
> > +      <customDisplay>{{velocity}}
> > +#set ($order = [
>
> Because the list is too long? Some are not appropriate? Or another reason?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phenotips/phenotips/pull/1698/files#r32121183>.
>